subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -74,6 +74,17 @@ config BT_LLL_VENDOR_OPENISA
 	help
 	  Use OpenISA Lower Link Layer implementation.
 
+# BT_CTLR_DF_SUPPORT is a wrapper for all DF features. Here we select features that are supported by
+# Zephyr's BLE Controller.
+config BT_CTLR_DF_SUPPORT
+	bool
```

Comment:
You do not need the `bool` here in this file,  but i guess Kconfig is not complaining the way it is now, may be without `bool` it may complain when `Kconfig.df` is not included (only `Kconfig.ll_sw_split` get included)?

Reply:
Updated, the type should be added in top definition of a KConfig entry. It may be skipped in places where it is extended.